requirements.txt

```diff
@@ -1,7 +1,8 @@
 pycurl
 peewee
 aiodns
-git+https://github.com/Rapptz/discord.py@rewrite#egg-discord.py[voice]
+git+https://github.com/Rapptz/discord.py[voice]
+discord.py
```

Comment:
We don't need this included twice, discord.py[voice] should be sufficient, since it looks like the rewrite has been released to pypy now.